src/style.css

```diff
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: "Avenir-Book";
+  font-weight: 300;
+  color: #3d4e61;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  font-weight: 300;
+  color: #3d4e61;
+  src: url("./fonts/Avenir-Heavy.ttf");
+
+}
+
+.form_field {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.icon_big {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.field_big {
+  box-sizing: border-box;
+  font-size: 16px;
+  line-height: 22px;
+  width: 100%;
+  height: 70px;
```

Comment:
do not hardcode height, use vertical paddings

Reply:
When i applied paddings, not a single test went, i don't undeestand why